### PR TITLE
Flowcheck / schema2* loader

### PR DESCRIPTION
**Pull request by alxrsngrtn**
- Using a shell utility to promote consistent module loading in dataflow and schema2base
- Converted string interpolation to concatenation.

### CODE REVIEW

File: src/dataflow/cli/flowcheck.ts [hunk 1]
```diff
@@ -9,25 +9,24 @@
...
     console.error('Usage: flowcheck <manifest files>');
     process.exit(1);
   }
 
-  const loader = new Loader();
   for (const filename of filenames) {
-    console.log('Checking file ' + filename);
-    const manifest = await Manifest.load(filename, loader);
+    console.log(`Checking file ${filename}`);
+    const manifest = await Utils.parse(`import '${filename}'`);
```
Comment (csilvestrini) [inline src/dataflow/cli/flowcheck.ts]: FMI: why this change? What does Utils.parse do that Manifest.load doesn't?

Comment (cromwellian) [reply src/dataflow/cli/flowcheck.ts]: According to Scott, it allows macros to work.
